program.go

```diff
@@ -55,3 +55,83 @@ func (p *Program) Streams() (ss []*Stream) {
 	}
 	return
 }
+
+// https://ffmpeg.org/doxygen/7.0/structAVProgram.html#a8c87564167b87f54be9171778d51fe49
+func (p *Program) Flags() int {
```

Comment:
I don't want this lib to provide flags as `int` but specific typed flags instead (check out [here](https://github.com/asticode/go-astiav/blob/master/codec_context.go#L134) for instance).

Here's how you should generate your new flags:
- you create `ProgramFlag` in `program_flag.go` similar to [this](https://github.com/asticode/go-astiav/blob/master/codec_context_flag.go) with all the proper values
- in `internal/cmd/flags/main.go` you add `Program` alphabetically ordered in the `list` variable
- you run `make generate-flags` at the root of the project
- that should have generated a struct `ProgramFlags`
- you use `ProgramFlags` instead of  `int` in `Flags()` and `SetFlags()`

Reply:
I can not find any enum program flags and any informations with AVProgram in ffmpeg source code.
I dont know how to set program.flags, maybe should be remove SetFlags()?

Reply:
You don't need to set flags in your workflow? Which methods are you really going to use in your workflow? I'm not looking to add all methods if you don't need them for now

Reply:
delete SetFlags function.i think it is useless.

Reply:
Can you remove `Flags()` as well if you're not using it?

Reply:
Had removed `Flags()` and `SetFlags()`.